### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by luh-99** [luh-99/School-Stuff#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shragsir** [comment]
Are you still working on this? I think it's been 6 updates since you forked my repo. I found this around an hour ago, and feel kind of bad about the lack of interest from other people. I'm right here if you want any help or want to partner for this. If you respond, it might be a little bit before I get back to you. Serious middle school stuff going down.

**Reply by luh-99** [reply]
> Are you still working on this? I think it's been 6 updates since you forked my repo. I found this around an hour ago, and feel kind of bad about the lack of interest from other people. I'm right here if you want any help or want to partner for this. If you respond, it might be a little bit before I get back to you. Serious middle school stuff going down.

Yeah that sounds fine, I’m not able to regularly update because of HS. The time we are in class, the time between classes, and during passing period is very limited. And I’m also not allowed to use my computer and phone at home.

**Reply by shragsir** [reply]
Literally, same.

**Comment by shragsir** [comment]
Also, here's my Email;
[click me!](mailto:shrashep@outlook.com)
